### ISSUE TITLE
How to add golang code?

### ISSUE THREAD

**Issue by cupen** [AsynkronIT/protoactor-bootcamp#4]
I can write a script to generate `GO` directory, then fill some files consists by the document of `C# version` and go code. 
```
ENG/C#
ENG/GO

RUS/C#
RUS/GO
```
But it will make a lot of repetitive text. Maybe it's not a good solution.

**Comment by cupen** [comment]
I would like this interactive tab, but it is more `html`.  :smiley:  Any suggestions?
https://proto.actor/docs/hello-world/

**Comment by cupen** [comment]
How about this?  Add some links of other languages to your `C# version`.
```
examples
├── unit-1
│   └── lesson-1
│       ├── csharp
│       │   ├── Program.cs
│       │   └── project.xml
│       ├── golang
│       │   ├── main.go
│       │   └── other.go
│       └── python
│           └── main.py
└── unit-2
    └── lesson-1
        ├── csharp
        │   ├── Program.cs
        │   └── project.xml
        ├── golang
        │   ├── main.go
        │   └── other.go
        └── python
            └── main.py
```



e.g.: 
- - -
Let's look at an example of what might look like a message class written in `C#`. [other languages](https://github.com/AsynkronIT/protoactor-bootcamp/issues/4)
```c#
private class ExampleMessage
{
    public int CustomerID { get; }

    public Hello(int customerId)
    {
        CustomerID = customerId;
    }
}
```
- - -
@Valentin-Miroshnichenko  Any suggestions?

**Comment by rogeralsing** [comment]
@cupen on the proto.actor website, we have code snippets in Tabs, where users can switch language.
I think that would be the easiest in the end to maintain, so there is only one copy of the bootcamp itself, but with multiple snippets. (and example code folders)

We have tried to maintain different language versions of docs in the past and its very hard to keep all bits in sync

**Comment by cupen** [comment]
@rogeralsing  Yes, I agree with you.   `Tabs` is really good,  but it's not supported by [github flavored markdown]. 

[github flavored markdown]: https://github.github.com/gfm/
